src/xcb/xconn.rs

```diff
@@ -70,7 +70,7 @@ impl XcbConnection {
     }
 
     fn window_has_type_in(&self, id: WinId, win_types: &[u32]) -> bool {
-        if let Ok(atom) = self.api.get_atom_prop(id, Atom::NetWmWindowType) {
+        if let Ok(atom) = self.api.get_atom_prop(id, "_NET_WM_WINDOW_TYPE") {
```

Comment:
can this be changed to `Atom::NetWmWindowType.as_ref()` like the rest of the code base please: I want to avoid any hard coded `&str` literals for known atom names.

Reply:
Done. I amended that into the commit.

Yeah, I was not totally happy with that solution either, but obviously didn't think enough about it. Sorry.

Reply:
Thanks! If you can rebase off of current develop I'll get this merged :+1:

Reply:
Done